cli/flox/doc/manifest.toml.md

```diff
@@ -0,0 +1,310 @@
+---
+title: MANIFEST.TOML
+section: 1
+header: "Flox User Manuals"
+...
+
+
+# NAME
+
+manifest.toml - declarative environment configuration format
+
+# SYNOPSIS
+
+The `manifest.toml` file is a declarative format for specifying the packages
+installed in an environment,
+environment variables to make available to the environment,
+a shell script to run upon activation of the environment,
+and other options for how the environment should behave.
+
+# DESCRIPTION
+
+Flox environments come with a declarative manifest in TOML format.
+An environment can be reproduced from this one file.
+The file is divided into just a few sections that are represented as TOML
+tables:
+
+- `[install]`
+- `[vars]`
+- `[hook]`
+- `[options]`
+
+## `[install]`
+
+The `[install]` table is the core of the environment,
+specifying which packages you'd like installed in the environment.
+An example of the `[install]` table is shown below:
+
+```toml
+[install]
+ripgrep.pkg-path = "ripgrep"
+pip.pkg-path = "python310Packages.pip"
+```
+
+Since this is TOML, an equivalent way of writing this would be
+```toml
+[install]
+ripgrep = { pkg-path = "ripgrep" }
+pip = { pkg-path = "python310Packages.pip" }
+```
+Flox will use the first format by default when automatically editing
+the manifest.
```

Comment:
I don't think so. I think most people won't be doing successive edits like that and are more likely to use the dotted notation anyway. Ultimately the user can decide whatever format they want since they're equivalent and it's personal preference.